library/Core.cpp

```diff
@@ -35,7 +35,6 @@ distribution.
 #include <forward_list>
 #include <type_traits>
 #include <cstdarg>
-using namespace std;
```

Comment:
while I get that `using namespace std;` is bad, I wouldn't have complained if we kept `using std::string` and `using std::vector`. That's pretty commonplace in our code, and I don't see anything wrong with it.

Reply:
i did consider doing that, but have mixed feelings here